src/lib/utils/thorchain/hooks/useSendThorTx.tsx

```diff
@@ -70,6 +72,7 @@ export const useSendThorTx = ({
   fromAddress,
   thorfiAction,
   isSubmitting = false,
+  onSend,
```

Comment:
hesitant about adding this at this time unless there is a better reason than just setting the txid which we pass back and can do in the consumer to keep things a bit more isolated

Reply:
Removed in the PR stacked on top of this one! https://github.com/shapeshift/web/pull/6729/commits/010ab3bc0e7abec7051e96a79b8be886d35a514d